Dump/run_otg_main_script

```diff
@@ -8,16 +8,12 @@ echo "run_filter_scripts.py done"
 /data/mta/Script/Dumps/Scripts/run_otg_proc.py
 echo "run_otg_proc.py done"
 
+/data/mta/Script/Dumps/Scripts/plot_otg_move.py
+echo "plot_otg_move.py done"
+
 /data/mta/Script/Dumps/Scripts/update_rdb_and_html.py
 echo "update_rdb_and_html.py done"
 
 /data/mta/Script/Dumps/Scripts/plot_otg_data.py
 echo "plot_otg_data.py done"
 
-/data/mta/Script/Dumps/Scripts/check_prev_getnrt_process.py
```

Comment:
Why has this processing been removed?

Reply:
The Dumps script currently operates without this python script. This is missing due to my first commit of updating our github to live running. I'll clarify this in future PR's by separating those "bring to live running' commits out separately as suggested to avoid confusion.

Reply:
Why have these three scripts being disabled in the current version and what is their function?

check_prev_getnrt_process.py
getnrt_control.py
update_rdb.py

Reply:
This deletion is due to copying the current running version in /data/mta/Script into the github version. None of these scripts run currently. In terms of their original purpose, 
check_prev_getnrt_processes ensure that the getnrt process isn't stalling from a previous script run. Since getnrt is now an executable called by the update_dea_rdb.py script, a stall will cause that python script lock file to be addressed, which is part of implementing MTA coding standards.
getnrt_control.py is for running a version of getnrt on SIM data, which is not handled by another set of scripts.
update_rdb.py is for updating MTA owned RDB files in Dataseeker. Of the ones handle by this script, the OTG moves RDB have been transferred to run on update_rdb_and_html.py, while the others have remained turned off since october 2022 according to their file modification time in /data/mta/DataSeeker/data/repository.